src/main/java/org/eclipse/tractusx/ssi/lib/jwt/SignedJwtFactory.java

```diff
@@ -116,10 +117,16 @@ private static SignedJWT createSignedES256Jwt(
 
       var algorithm = JWSAlgorithm.EdDSA;
       var type = JOSEObjectType.JWT;
+
+      // https://w3c.github.io/vc-jws-2020/#json-web-signature-2020
+      var crit = new HashSet<String>();
+      crit.add("b64");
+
       var header =
           new JWSHeader(
```

Comment:
Please use the `JWSHeader.Builder` to create the header, as it is more expressive.